src/Arcus.EventGrid.Security/Arcus.EventGrid.Security.csproj

```diff
@@ -19,7 +19,7 @@
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' != 'netcoreapp3.1'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.1'">
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.0.1" />
```

Comment:
Are we sure that this is still correct ?  Do we need to include this package when we're targetting 2.1 ?

Reply:
Well, the packages for .NET Core 3.1 & .NET 6 are not compatible with .NET standard 2.1, so I would say, yes. 🤔

Reply:
I'll check again.

Reply:
Oh, I see. This project is actually in its entirety deprecated. We have moved this to `Arcus.EventGrid.WebApi.Security`. So it's for backwards compatibility reasons.

Reply:
Arcus.EventGrid.Security is deprecated you mean ?
How do we cope with Arcus projects that we deprecate ?  How long do we keep them around ?  Do we put a warning on the readme that it is deprecated ?

Reply:
> Arcus.EventGrid.Security is deprecated you mean ?

Yes, should not be used for adding new features and maintaining.

> How do we cope with Arcus projects that we deprecate ? 
> How long do we keep them around ? Do we put a warning on the readme that it is deprecated ?

Same as deprecated funtionality, keep around till the next major, and then we can delete them. We should probably add indeed some extra deprecation information on all these types. I'll add an issue for this: #204

Reply:
If it's deprecated, do we need to make the changes that have been made in this PR then ?

Reply:
Ah, you're right. I think we can remove the .NET 6 part here. Good catch!

Reply:
I removed the .NET 6 part, but we still need to support .netstandard2.1 here, as the dependent `Arcus.EventGrid` project is .netstandard2.1.

Reply:
I also marked the functionality as obsolete and set the NuGet package as 'deprecated'. #206 #205

Reply:
Sorry, I didn't keep track of this one, but I think we can complete this one.